rfc081-release-cadence.md

```diff
@@ -41,6 +41,32 @@ potential hot fixes or follow-up.
 Major releases in April avoids releasing during winter holidays, summer
 vacations, ChefConf and Chef Summits.
 
+## Cookbook Support
+
+The latest version of community cookbooks are required to support only the latest 6
```

Comment:
Maybe say "encouraged to support at least the latest 6" since this is just a spec and we can't enforce adherence.